src/dparse/lexer.d

```diff
@@ -1786,8 +1786,11 @@ private pure nothrow @safe:
  */
 public auto byToken(ubyte[] range)
 {
+    import std.math : nextPow2;
```

Comment:
Well, the oldest version of Phobos that we're testing is the on`ldc` ships with - i.e. 2.073.1. Even GDC master now supports this. I think we can assume that nothing older than that it's supported.